Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk/WebKit/CMBaseObjectSPI.h

```diff
@@ -36,7 +36,9 @@ extern "C" {
 enum {
     kCMBaseObject_ClassVersion_1 = 1,
     kCMBaseObject_ClassVersion_2 = 2,
-    kCMBaseObject_ClassVersion_3 = 3
+    kCMBaseObject_ClassVersion_3 = 3,
+    kCMBaseObject_ClassVersion_4 = 4,
+    kCMBaseObject_ClassVersion_5 = 5
```

Comment:
I can leave out this change; was just syncing to an internal header to add versions.